tests/core/test_model.py

```diff
@@ -3759,3 +3809,28 @@ def test_this_model() -> None:
 
     assert model.render_pre_statements()[0].sql() == """COPY "db"."table" TO 'a'"""
     assert model.render_post_statements()[0].sql() == """COPY "db"."table" TO 'b'"""
+
+
+def test_macros_in_model_statement(sushi_context, assert_exp_eq):
+    expressions = d.parse(
+        """
+        MODEL (
+            name @{gateway}.test_model,
```

Comment:
maybe just make this a bit more complex,

name @{gate_way}_@{gate_way}.test_model

Reply:
There's a combo scenario in the other test. Or did you mean to test something else?

Reply:
Updated